### ISSUE TITLE
Add pagination to REST publish/subscribe

### ISSUE THREAD

**Issue by dsuch** [zatosource/zato#680]
In ticket #677 pagination is being added to all listings in web-admin. However, since REST publish/subscribe is backed by Redis currently, and pagination requires an SQL backend, this feature of pagination cannot be added to REST pub/sub.

On the other hand, an SQL DB will be used for REST pub/sub in Zato 3.0 anyway so this ticket is just a remainder to include pagination once the port from Redis to SQL is done.

**Comment by dsuch** [comment]
Closing in favour of #755.
